docs/src/content/docs/de/guides/sidebar.mdx

```diff
@@ -32,17 +32,15 @@ Wird die folgende Seitenleiste automatisch generiert:
 <SidebarPreview
 	config={[
 		{
-			label: 'Anleitungen',
+			label: 'Konstellationen',
 			items: [
-				{ label: 'Komponenten', link: '/guides/components/' },
-				{ label: 'Internationalisierung (i18n)', link: '/guides/i18n/' },
+				{ label: 'Andromeda', link: '' },
+				{ label: 'Jäger', link: '' },
```

Comment:
Just to double check, is "Jäger" expected here? Asking because Wikipedia [seems](https://de.wikipedia.org/wiki/Orion_(Sternbild)) to just use Orion?

Reply:
I looked up the constellation but seems that I have mixed up something, my bad...